pkg/chaosdaemon/netem.go

```diff
@@ -94,10 +94,41 @@ func Cancel(netem *pb.Netem, pid uint32) error {
 		},
 	}
 
+	exist, err := qdiscExists(netemQdisc, handle, link)
+	if err != nil {
+		log.Error(err, "failed to check qdisc", "qdisc", netemQdisc, "link", link)
+		return err
+	}
+
+	if !exist {
+		log.Error(nil, "qdisc not exists", "qdisc", netemQdisc)
```

Comment:
I recommend to add a warning message on the log, eg. `it maybe had been removed by accident, or there exists a bug on chaos-mesh`, because you return nil on this case.